scripts/portView.js

```diff
@@ -0,0 +1,76 @@
+/*portView.js is a file dedicated to handling the viewing functionality of the portfolio articles and list*/
+
+var portArticleView = {};
+
+portArticleView.handlePreview = function(){
+  $('.portPreview').hide();
+
+  $('#portList').on('click', 'a' , function(e){
+    /*TODO: Add conditional statement to prevent opening another port article if one is already open. */
+    e.preventDefault();
+    var title = $(this).text();
```

Comment:
if your only gonna use the variable title once then just put $(this).text() in the one play you need it. Unless you want it to know what $(this).text() is automatically without sifting through the code.